static/javascript/launch.js

```diff
@@ -152,54 +165,72 @@ function clearSurveyMetadataFields() {
   showMetadataAccordion("sds", false);
 }
 
-function setSurveyType(event) {
-  localStorage.setItem("survey_type", event.value);
-  setLaunchType("remote");
+function validateRemoteSection() {
```

Comment:
Maybe we should focus on what it does exactly so something more like  `setLoadMetadataButtons()`?

Reply:
To me `setLoadMetadataButtons` isn't that descriptive of what the function actually does, Ive updated it to `enableDisableLoadMetadataButton` although I do think this is quite a long function name

Reply:
Maybe we should use `toggle`? Sort of confusing since enable/disable are opposites.